testdata/kfunc.c

```diff
@@ -2,9 +2,11 @@
 
 char __license[] __section("license") = "Dual MIT/GPL";
 
+extern void bpf_testmod_test_mod_kfunc(int) __ksym;
```

Comment:
Could you split this into a separate file `kfunc-kmod.c` plus a separate test `TestKfuncKmod` or similar? Then we still test regular kfuncs on systems that don't have bpf_testmod.

Reply:
Fixed :ballot_box_with_check: